src/constants.js

```diff
@@ -420,3 +420,4 @@ export const KEYS = {
 
 /* ----------------- Other ----------------------- */
 export const ONE_HOUR_MS = 3600000; // 60 * 60 * 1000
+export const BASE_SELECT_FIELD_OFFSET_TOP_PADDING = 75;
```

Comment:
Can we add a comment explaining the significance of the magic number of `75`? Also, this is a very specific and narrowly-used constant, so it should probably live in the component itself rather than in the global constants file.